flutter/lib/src/navigation/display_strategy_evaluator.dart

```diff
@@ -0,0 +1,63 @@
+import 'dart:async';
+
+import 'package:meta/meta.dart';
+
+@internal
+class DisplayStrategyEvaluator {
```

Comment:
`l`: This class name should reflect that it deals with TTID. `DisplayStrategyEvaluator` doesn't reveal that. Let's have a chat on that in our 1on1.